ImperatorToCK3/Outputter/SuccessionTriggersOutputter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ImperatorToCK3.CK3.Titles;
+using System.IO;
+
+namespace ImperatorToCK3.Outputter {
+	public static class SuccessionTriggersOutputter {
+		public static void OutputSuccessionTriggers(string outputModName, LandedTitles landedTitles) {
+			var outputPath = Path.Combine("output", outputModName, "common/scripted_triggers/00_succession_triggers.txt");
+
+			using var outputStream = File.OpenWrite(outputPath);
+			using var output = new StreamWriter(outputStream, System.Text.Encoding.UTF8);
+
+			List<string> primogenitureTitles = new List<string>();
+			List<string> seniorityTitles = new List<string>();
+
+			foreach (var landedTitle in landedTitles) {
+				if (landedTitle.Value.DeFactoLiege == null) {					
+					if (landedTitle.Value.SuccessionLaws.Contains("single_heir_succession_law")) {
+						primogenitureTitles.Add(landedTitle.Key);
+					}
+					if (landedTitle.Value.SuccessionLaws.Contains("single_heir_dynasty_house")) {
+						seniorityTitles.Add(landedTitle.Key);
+					}
+				}
+			}
```

Comment:
You can deconstruct dictionary entries like this:
```suggestion
			foreach (var (name, title) in landedTitles) {
				if (title.DeFactoLiege == null) {					
					if (title.SuccessionLaws.Contains("single_heir_succession_law")) {
						primogenitureTitles.Add(name);
					}
					if (title.SuccessionLaws.Contains("single_heir_dynasty_house")) {
						seniorityTitles.Add(name);
					}
				}
			}
```